lib/tasks/migrate_to_pg.rake

```diff
@@ -92,3 +107,24 @@ def migrate_model(pg_model, mongo_model, transform = nil)
 
   puts "#{pg_model.count} #{pg_model} migrated to pg"
 end
+
+def migrate_fulfillment(pt_model, mongo_pt_model, pg_model, mongo_model, relation, parent_relation, transform)
+  attributes = pg_model.attribute_names
+  mongo_pt_model.collection.find.batch_size(100).each do |doc|
+    mongo_obj = doc[relation]
+
+    pg_attrs = transform.call(mongo_obj.slice(*attributes), mongo_obj, doc)
+    pg_obj = pg_model.create! pg_attrs
+    if mongo_obj['created_by_id'].present?
+      pg_obj.versions.first.update whodunnit: User.find_by(mongo_id: mongo_obj['created_by_id'].to_s)&.id
+    end
+
+    # Check
+    if pg_model.count != mongo_model.count
+      'PG and mongo counts are in disagreement, aborting'
```

Comment:
Is this rake task magic?

Reply:
nah, `pg_model` and `mongo_model` should be the classes in question. `pg_model = Fulfillment` and `mongo_model = MongoFulfillment` for ex.

Reply:
I meant line 124 just being a string!

Reply:
lol my mistake, sorry! I assume you're asking whether or not this should `puts` and it def should. thank you!

Reply:
```suggestion
      raise 'PG and mongo counts are in disagreement, aborting'
```

Reply:
(puts, raise, whatever.)